src/app/incident/incident-register/incident-register.component.spec.ts

```
@@ -0,0 +1,122 @@
+import { of, throwError } from 'rxjs';
+import { faker } from '@faker-js/faker';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations'; // Import this module
+
+export interface IncidentResponse {
+  client_id: string;
+  name: string;
+  channel: string;
+  reported_by: string;
+  created_by: string;
+  description: string;
+}
```

Biome

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)